index.html

```diff
@@ -1,20 +1,197 @@
+    <!-- <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js" -->
+        <!-- integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW" crossorigin="anonymous"> -->
+        <!-- </script> -->
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css" title="Default Styles" media="screen">
+    <script src="main.js"></script>
 
-    <title>Responsive grid project</title>
 </head>
 
 <body>
+        <div id="body-wrapper">
+            <!-- header -->
+            
+            <!-- <header class="index-header-class"> -->
+            
+                <!-- Try -->
+                <nav class=" index-header navbar  navbar-expand-lg navbar-light bg-light d-flex justify-content-space-between align-items-start">
+                    <a class="navbar-brand " href="index.html"> 
+                        <div class="row">
+                            <div class="col-1">
+                                <div style="color:blue;font-size:120%">&#9632;</div>
+                            </div>
+                            <div class="col">
+                                <b>Benyam Worku</b>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col">
+                                Trainee Developer
+                            </div>
+                        </div>
+                        </a>
+                    <button class="navbar-toggler " type="button" data-toggle="collapse" data-target="#myNav"
+                        aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse " id="myNav">
+                        <div class="navbar-nav ml-auto  ">
+                            <a class="nav-item nav-link active" href="index.html" style="color:blue">ABOUT ME <span class="sr-only">(current)</span></a>
+                            <a class="nav-item nav-link" href="resume.html">RESUME</a>
+                            <a class="nav-item nav-link" href="projects.html">PROJECTS</a>
+                            <a class="nav-item nav-link" href="contact.html">CONTACT</a>
+                        </div>
+                    </div>
+                </nav>
+                
+            <!-- </header>  -->
+            <!-- profile card :photo,buttons,social-media  -->
+            <div class=" profile-card container ">
+                <div class="photo-wrapper row">
+                    <div class="offset-3 col-6">
+                        <img src="img/Benny_photo.png" alt="Ben's photo">
+                    </div>
+                </div>
+                <div class="name-wrapper row">
+                    <div class="offset-4 col-5">
```

Comment:
Currently, your name is not exactly centered. Using ``col`` class here allows to have your name centered